apps/strapi-cms/.env.default

```diff
@@ -49,4 +49,7 @@ DEPLOY_WEBSITE_TARGET_BRANCH=
 GOOGLE_OAUTH_CLIENT_ID=
 GOOGLE_OAUTH_CLIENT_SECRET=
 GOOGLE_OAUTH_REDIRECT_URI=
-GOOGLE_GSUITE_HD=
\ No newline at end of file
+GOOGLE_GSUITE_HD=
+
+## IMAGES CDN URL
+CDN_URL=
```

Comment:
I see that to for each env variables corrisponde some changes in `apps/infrastructure`. I think that also for this var those changes are necessary you can take `GOOGLE_GSUITE_HD` var as an example i suppose.